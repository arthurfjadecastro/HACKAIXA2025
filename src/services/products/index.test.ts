import * as productsIndex from './index';

describe('Products Services Index', () => {
  it('should export products modules', () => {
    expect(productsIndex).toBeDefined();
    expect(typeof productsIndex).toBe('object');
  });

  it('should re-export API, data and types modules', () => {
    // Verifica se mÃ³dulos foram exportados sem erros
    expect(() => {
      const exported = { ...productsIndex };
      expect(exported).toBeDefined();
    }).not.toThrow();
  });

  it('should match snapshot', () => {
    expect(Object.keys(productsIndex).sort()).toMatchSnapshot();
  });
});
