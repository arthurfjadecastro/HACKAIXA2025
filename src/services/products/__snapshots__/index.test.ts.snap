// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Products Services Index should match snapshot 1`] = `
[
  "ProductDataService",
  "productService",
]
`;
